README.md

```diff
@@ -1,207 +1,237 @@
+    ```bash
+    $ export SIGNALFX_SERVICE_NAME="your_app"
+    $ export SIGNALFX_AGENT_HOST="localhost"
+    $ export SIGNALFX_TRACING_ENABLED="true"
+    $ export SIGNALFX_SPAN_TAGS="key1:val1,key2:val2"
+    $ export SIGNALFX_DB_STATEMENT_MAX_LENGTH=1024
+    $ export SIGNALFX_RECORDED_VALUE_MAX_LENGTH=12288
+    $ export SIGNALFX_TRACE_ANNOTATED_METHOD_BLACKLIST=null
+    $ export SIGNALFX_MAX_SPANS_PER_TRACE=0
+    $ export SIGNALFX_MAX_CONTINUATION_DEPTH=100
+    ```
+1. Include the agent in your Java application: 
+    ```bash
+    $ java -javaagent:path/to/signalfx-tracing.jar -jar app.jar
+    ```
+
+## Troubleshoot the SignalFx Java Agent
+
+Enable debug logging for troubleshooting assistance. Set the
 `datadog.slf4j.simpleLogger.defaultLogLevel=debug` property at runtime.  These logs
-are extremely verbose.**
+are extremely verbose.
 
-```bash
-$ export SIGNALFX_SERVICE_NAME=my-app
-$ export SIGNALFX_SPAN_TAGS="MyTag:MyValue"
-$ java -javaagent:signalfx-tracing.jar <rest of command-line arguments...>
-```
-
-The standard invocation assumes that you have a [locally deployed SignalFx Smart
-Agent](https://docs.signalfx.com/en/latest/apm/apm-deployment/smart-agent.html)
-accessible on `localhost`. If you want to send traces directly to the Smart
-Gateway or if your Smart Agent is only accessible on a different host (e.g. in a
-containerized environment), set the trace host via the `signalfx.agent.host`
-property or its corresponding `SIGNALFX_AGENT_HOST` environment variable to the
-appropriate value. You can override the entire trace target URL with the
-`signalfx.endpoint.url` property or its corresponding `SIGNALFX_ENDPOINT_URL`
-environment variable.
-
-By default, the Java Agent's tracer will have constant sampling (100% chance of
-tracing) and report every span to the Agent. Where applicable, context
-propagation will be done via [B3
-headers](https://github.com/openzipkin/b3-propagation).
-
-See [our example applications that use
-this utility](https://github.com/signalfx/tracing-examples/tree/master/signalfx-tracing/signalfx-java-tracing) for
-more details on how to use and configure the SignalFx Java Agent.
+## Manually instrument a Java application
 
-### Trace Annotation
+You can use the OpenTracing `GlobalTracer` or a `Trace` annotation to manually
+instrument your Java application.
 
-In cases where you'd like to gain some of the benefits of custom instrumentation without
-using the OpenTracing `GlobalTracer` and API directly, we offer a `@Trace` annotation you
-can use where desired:
+### Configure the OpenTracing `GlobalTracer`
 
+The SignalFx Java Agent configures an OpenTracing-compatible tracer
+to capture and export trace spans. It registers this tracer as the OpenTracing
+`GlobalTracer` to easily enable custom instrumentation throughout your
+application:
 ```java
-import com.signalfx.tracing.api.Trace;
+import io.opentracing.util.GlobalTracer;
```

Comment:
@rmfitzpatrick is this correct?

Reply:
Yes, but `import io.opentracing.*;` is also needed for this snippet for Tracer, Scope, and Span.